src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -1429,9 +1428,6 @@ public override IOperation VisitMethodBodyOperation(IMethodBodyOperation operati
             var member = operation.SemanticModel.GetDeclaredSymbol(operation.Syntax);
             ImmutableArray<IParameterSymbol> parameters = member switch
             {
-                null => ImmutableArray<IParameterSymbol>.Empty,
-                IFieldSymbol _ => ImmutableArray<IParameterSymbol>.Empty,
-                IParameterSymbol parameterMember => ImmutableArray.Create(parameterMember),
                 IMethodSymbol method => method.Parameters,
                 _ => throw ExceptionUtilities.UnexpectedValue(member)
             };
```

Comment:
If this works, consider collapsing to a cast: `var parameters = ((IMethodSymbol)member).Parameters;`

Reply:
Will do if removal doesn't cause test failures---waiting and seeing.

Reply:
Done